worker/lib/workers/testbot.ts

```diff
@@ -96,6 +96,15 @@ class TestBotWorker extends EventEmitter implements Leviathan.Worker {
 		console.log('Vout=', await this.hatBoard.readVout());
 	}
 
+	public async diagnostics() {
+		return {
+			worker: "testbot",
```

Comment:
Up for some discussion about this diagnostics endpoint -

At the moment, we actually have an endpoint in the worker called "/contract": https://github.com/balena-os/leviathan/blob/master/worker/lib/index.ts#L58 it returns a json object with some information in it. This includes the worker type.

So we have some overlap between contract/diagnostic. We can either roll them in to one endpoint, or keep them seperate. If we keep them separate, maybe the answer is to have static information (worker type, hardware connected etc) to the contract, and dynamic information such as output voltage in the diagnostics endpoint. What do you think?

Reply:
I agree, implemented.